### PR TITLE
DOCS-3647 Adds Synthetics and RUM Application Testing Coverage Dashboards Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds two doc pages and screenshots for Synthetics & RUM Testing Coverage dashboards.

### Motivation

DOCS-3647

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: RUM Application Testing Coverage Dashboard
```
Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:2]: I believe we can drop the Synthetics or RUM in the title and stay with "Application Testing Coverage Dashboard"

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  tag: 'Documentation'
+  text: 'Learn about RUM & Session Replay'
+- link: '/synthetics/browser_tests'
+  tag: 'Documentation'
+  text: 'Learn about Synthetic browser tests'
+---
+
+## Overview
+
+The [application testing coverage dashboard][1] uses data collected from [RUM & Session Replay][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application. For more information, see [Explore RUM & Session Replay in Synthetics][4].
```
Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:16]: Data comes from RUM only. no need for Session replay. I think it is important to make the difference as Session replay is made as a premium https://www.datadoghq.com/pricing/?_gl=1*cx28xz*_gcl_aw*R0NMLjE2NTMzOTMyMTUuQ2owS0NRandoTEtVQmhEaUFSSXNBTWFUTG5FOUlZaWRURVgxRmc5SGNnNmpCVkYxR0I1YVRrN1ZGU3pWcXphN0FPZDNUMHh2bjdyRzBnZ2FBdlljRUFMd193Y0I.*_ga*MTIyOTE5NDU0LjE2MjM2ODExMzI.*_ga_KN80RDFSQK*MTY1NjUxNDExOC45MTIuMS4xNjU2NTE0MTg4LjYw&product=real-user-monitoring--session-replay#real-user-monitoring--session-replay

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+You can use this dashboard to better understand what is and what is not tested in your monitored applications, and identify the most popular sections of your application to continuously monitor. You are alerted when key user journeys are negatively impacted by a code change. It shows:
```
Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:18]:
```suggestion
You can use this dashboard to better understand what is and what is not tested in your monitored applications, and identify the most popular sections of your application to continuously monitor. By adding the most popular sections to an existing Synthetics browser tests or creating a new one, you are alerted when key user journeys are negatively impacted by a code change. It shows:
```

File: content/en/synthetics/dashboards/_index.md [hunk 4]
```diff
@@ -15,19 +15,20 @@ further_reading:
...
+Access your Synthetics dashboards by filtering for `Synthetics` in the search query of the [**Dashboard List**][2] or by clicking on the dropdown menu under [**Dashboards**][3] on the [Synthetic Monitoring homepage][4].
 
 {{< img src="synthetics/dashboards/synthetic_tests_dashboards.png" alt="Synthetic Monitoring Dashboards" style="width:100%;">}}
 
 {{< whatsnext desc="You can explore the following out-of-the-box Synthetics dashboards:" >}}
   {{< nextlink href="/synthetics/dashboards/api_test" >}}<u>API Test Performance</u>: Monitor your endpoints and services. {{< /nextlink >}}
   {{< nextlink href="/synthetics/dashboards/browser_test" >}}<u>Browser Test Performance</u>: View your browser tests' web performance, insights into third-party providers, and Core Web Vitals. {{< /nextlink >}}
+    {{< nextlink href="/synthetics/dashboards/test_summary" >}}<u>Testing Coverage</u>: Evaluate your browser tests' application coverage and identify popular elements in your application to track using RUM and Synthetics data. {{< /nextlink >}}
```
Comment (hestonhoffman) [inline content/en/synthetics/dashboards/_index.md:25]:
```suggestion
    {{< nextlink href="/synthetics/dashboards/testing_coverage" >}}<u>Testing Coverage</u>: Evaluate your browser tests' application coverage and identify popular elements in your application to track using RUM and Synthetics data. {{< /nextlink >}}
```

**Review by HugoPct — CHANGES_REQUESTED** [review]

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  tag: 'Documentation'
+  text: 'Learn about Synthetic browser tests'
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM & Session Replay'
+---
+
+## Overview
+
+The [application testing coverage dashboard][1] uses data collected from [RUM & Session Replay][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application.
```
Comment (HugoPct) [inline content/en/synthetics/dashboards/testing_coverage.md:16]:
```suggestion
The [application testing coverage dashboard][1] uses data collected from [RUM][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application. 
```

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  text: 'Learn about RUM & Session Replay'
+---
+
+## Overview
+
+The [application testing coverage dashboard][1] uses data collected from [RUM & Session Replay][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application. 
+
+You can use this dashboard to build a more complete, accurate testing suite with information about the following topics:
+
+- A list of top web pages linked to RUM views
```
Comment (HugoPct) [inline content/en/synthetics/dashboards/testing_coverage.md:20]:
```suggestion
- A list of top visited web pages
```

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- The percentage of tested RUM actions
+- The number of tested and total actions
+- The number of browser tests covering actions
+- The number of real user interactions 
+
+It shows:
+
+- **Untested actions**: Explore the most popular untested user actions with the count of real user interactions and the number of actions covered in browser tests.
+- **Top Synthetic browser tests**: Analyze the the list of browser tests with the count of tested RUM actions.
```
Comment (HugoPct) [inline content/en/synthetics/dashboards/testing_coverage.md:29]: here the top 2 things of the dashboard are:
- % of tested actions
- Untested actions.

I think we can remove the **Top Syn BT** mention as it brings less value.

Example of a sentence for % of tested actions
- **Coverage of tested actions**: Get at a glance the overall coverage of your application.

(we can do this change for both guides)

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+By testing the most popular sections of your application through Synthetic browser tests, you are alerted when key user journeys in your application are negatively impacted by a code change. You can run tests [directly in your CI/CD pipelines][4] to ensure no regressions occur before releasing code in production. 
+
+To add a browser test for a top view or untested action, click on a view with a low percentage of tested actions under **Top Views** or an action under **Untested Actions** and select **Create a Synthetics browser test** from the dropdown menu. Clicking **View RUM events** navigates you to the [RUM Explorer][5] with an autofilled search query for actions with a specific view name.
+
+The **Top Views** table lists the most popular web pages that users are interacting with, along with the number of real user interactions and the percentage of actions that are tested in browser tests. 
+
+## Explore custom actions
+
+Use the dropdown menus to customize the data types and view [custom actions][6] specific to your queried data.
```
Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:52]: "dropdown menus" are called "template variables" in dashboards
here is the documentation about it https://docs.datadoghq.com/dashboards/template_variables/

Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:52]:
```suggestion
Use the dropdown menus to customize the data types and filter out your queried data on [custom actions][6]. 
```

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+To add a browser test for a top view or untested action, click on a view with a low percentage of tested actions under **Top Views** or an action under **Untested Actions** and select **Create a Synthetics browser test** from the dropdown menu. Clicking **View RUM events** navigates you to the [RUM Explorer][6] with an autofilled search query for actions with a specific view name.
+
+The **Top Views** table lists the most popular web pages that users are interacting with, along with the number of real user interactions and the percentage of actions that are tested in browser tests. 
+
+## Explore custom actions
+
+Use the dropdown menus to customize the data types and view [custom actions][7] specific to your queried data. 
+
+Because you are instrumenting your code to define these custom actions, your definitions lead to more accurate results compared to a generic, generated **Click on Save** action.
```
Comment (HugoPct) [inline content/en/synthetics/dashboards/testing_coverage.md:56]:
```suggestion
We encourage the usage of custom actions. By their nature, custom actions are unique. They will offer more accurate coverage results than generated actions.
```

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/dash/integration/30697/synthetics---browser-test-performance
+[2]: /real_user_monitoring/browser/data_collected/
+[3]: /synthetics/browser_tests/
+[4]: /synthetics/cicd_integrations/
+[5]: /real_user_monitoring/explorer
+[6]: /real_user_monitoring/guide/send-rum-custom-actions/
```
Comment (alai97) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:65]:
```suggestion
[6]: /real_user_monitoring/guide/send-rum-custom-actions/
[7]: /dashboards/template_variables/
```

File: content/en/real_user_monitoring/dashboards/testing_coverage.md (hunk 8 above)
Comment (alai97) [reply content/en/real_user_monitoring/dashboards/testing_coverage.md:52]: TIL, thanks!

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/dash/integration/30697/synthetics---browser-test-performance
+[2]: /real_user_monitoring/browser/data_collected/
+[3]: /synthetics/browser_tests/
+[4]: /synthetics/metrics/
+[5]: /synthetics/cicd_integrations/
+[6]: /real_user_monitoring/explorer
+[7]: /real_user_monitoring/guide/send-rum-custom-actions/
```
Comment (alai97) [inline content/en/synthetics/dashboards/testing_coverage.md:68]:
```suggestion
[7]: /real_user_monitoring/guide/send-rum-custom-actions/
[8]: /dashboards/template_variables/
```

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+By adding the most popular sections of your application to an existing browser test or creating a browser test, you are alerted when key user journeys in your application are negatively impacted by a code change. You can run tests [directly in your CI/CD pipelines][5] to ensure no regressions occur before releasing code in production. 
+
+To add a browser test for a top view or untested action, click on a view with a low percentage of tested actions under **Top Views** or an action under **Untested Actions** and select **Create a Synthetics browser test** from the dropdown menu. Clicking **View RUM events** navigates you to the [RUM Explorer][6] with an autofilled search query for actions with a specific view name.
+
+The **Top Views** table lists the most popular web pages that users are interacting with, along with the number of real user interactions and the percentage of actions that are tested in browser tests. 
+
+## Explore custom actions
+
+Use the dropdown menus to customize the data types and view [custom actions][7] specific to your queried data.
```
Comment (alai97) [inline content/en/synthetics/dashboards/testing_coverage.md:54]:
```suggestion
Use the [template variables][8] to customize the data types and view [custom actions][7] specific to your queried data. 
```

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+To add a browser test for a top view or untested action, click on a view with a low percentage of tested actions under **Top Views** or an action under **Untested Actions** and select **Create a Synthetics browser test** from the dropdown menu. Clicking **View RUM events** navigates you to the [RUM Explorer][5] with an autofilled search query for actions with a specific view name.
+
+The **Top Views** table lists the most popular web pages that users are interacting with, along with the number of real user interactions and the percentage of actions that are tested in browser tests. 
+
+## Explore custom actions
+
+Use the dropdown menus to customize the data types and view [custom actions][6] specific to your queried data. 
+
+Because you are the one instrumenting your code to define these custom actions, your definitions result in more accurate results compared to a generic, generated **Click on Save** action.
```
Comment (alai97) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:54]:
```suggestion
Datadog recommends using custom actions. By default, custom actions are unique and offer more accurate coverage results compared to generated actions.
```

File: content/en/real_user_monitoring/dashboards/testing_coverage.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+For more information about the data displayed, see [RUM Browser Data Collected][2].
+
+## Manage browser tests
+
+In order to improve your testing coverage and better manage your browser tests, this dashboard can help you answer the following questions:
+
+- What actions are not being tested in your application?
+- What views are the most popular to your users? 
+- What views need more browser tests?
+- What percentage of browser tests are covering user actions?
```
Comment (HugoPct) [inline content/en/real_user_monitoring/dashboards/testing_coverage.md:42]:
```suggestion
- What browser tests are covering user actions? 
```

File: content/en/synthetics/dashboards/testing_coverage.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+By adding the most popular sections of your application to an existing browser test or creating a browser test, you are alerted when key user journeys in your application are negatively impacted by a code change. You can run tests [directly in your CI/CD pipelines][5] to ensure no regressions occur before releasing code in production. 
+
+To add a browser test for a top view or untested action, click on a view with a low percentage of tested actions under **Top Views** or an action under **Untested Actions** and select **Create a Synthetics browser test** from the dropdown menu. Clicking **View RUM events** navigates you to the [RUM Explorer][6] with an autofilled search query for actions with a specific view name.
+
+The **Top Views** table lists the most popular web pages that users are interacting with, along with the number of real user interactions and the percentage of actions that are tested in browser tests. 
+
+## Explore custom actions
+
+Use the [template variables][7] to customize the data types and view [custom actions][8] specific to your queried data.
```
Comment (HugoPct) [inline content/en/synthetics/dashboards/testing_coverage.md:54]:
```suggestion
Use the [template variables][7] to customize the data types and filter out your queried data on [custom actions][8].  
```

**Pull request merged**